src/main/java/jenkins/plugins/gerrit/GerritWebHookCrumbExclusion.java

```diff
@@ -0,0 +1,33 @@
+package jenkins.plugins.gerrit;
+
+import hudson.Extension;
+import hudson.security.csrf.CrumbExclusion;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Extension
+public class GerritWebHookCrumbExclusion extends CrumbExclusion {
```

Comment:
Cool, I was just disabling the x-site scripting check on Jenkins, which was a lot worse than just excluding it with this extension. Well done 👍